src/styles/utils/header.scss

```diff
@@ -0,0 +1,72 @@
+.head {
+  padding-left: 50px;
+  padding-right: 50px;
+  font-weight: 500;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &--flex {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+}
+
+a {
+  display: inline-block;
+  text-decoration: none;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: v.$col-nav-link;
+  transition: all 0.3s;
+}
+
+.head-nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    gap: 20px;
+
+    a {
+      padding-top: 22.5px;
+      padding-bottom: 22.5px;
+      color: v.$col-nav-link;
+    }
+
+    a:hover {
+      color: v.$col-nav-link-hover;
+    }
+  }
+}
+
+/* .head__item1 {
+}
+
+.head-logo-link {
+}
+
+.head__item2 {
+}
+
+ */
+
+/* стиль для активной ссылки */
+.is-active {
+  position: relative;
+  color: v.$color-active;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background: v.$color-active;
+}
+
+a[data-qa='nav-hover']:hover {
+  color: v.$color-active;
```

Comment:
Ensure that the variable `v.$color-active` is defined in your variables file. If it's not defined, this will cause an error during compilation.